docs/get-started/backup/on_demand_backups.md

```diff
@@ -0,0 +1,158 @@
+7. Go to the **Restores** tab to check the status of the restored backup.
+During restoring, PMM disables all the scheduled backup tasks for the current service. Remember to re-enable them manually after the restore.
+
+### Restore to a new environment manually
+
+1. Install MongoDB and Percona Backup for MongoDB. Pay attention to the versions. To minimize potential incompatibility, use the same versions that were used for taking backups.
+   For instructions, see the [PBM install documentation](https://docs.percona.com/percona-backup-mongodb/installation.html).
+2. Configure your environment:
+     - to restore on a new environment with the same number of hosts and same replica set name, make sure that the replica set name in your new destination cluster use the same name as that in the cluster that was backed up.<br/>
+      For more information, see [Restoring a backup into a new-environment](https://docs.percona.com/percona-backup-mongodb/usage/restore.html#restoring-a-backup-into-a-new-environment) in the PBM documentation. <br> 
+     - to restore **logical backups** to a new environment that has different replica set name, configure the name mapping between the source and target environments. <br/>
+      For the new environment you can either set the **PBM_REPLSET_REMAPPING** environment variable for pbm CLI, or use the `--replset-remapping` flag for PBM commands.
+
+        The mapping format is `<rsTarget>=<rsSource>`.
+
+        For example:
+
+        `$ export PBM_REPLSET_REMAPPING="targetRS=sourceRS"`
+
+        OR 
+
+        `$ pbm restore <timestamp> --replset-remapping="targetRS=sourceRS"`
+
+        For more information, see [Restoring into a replica set with a different name](https://docs.percona.com/percona-backup-mongodb/usage/restore.html#restoring-into-a-cluster-replica-set-with-a-different-name) in the PBM documentation.
+
+3. Make sure that Percona Backup for MongoDB configuration in the new environment points to the remote storage defined for the original environment, including the authentication credentials for object stores.
+
+    The easiest way to configure it is to create a config file, called, for example, `pbm_config.yaml`.
+
+    For this, you can either copy the config from the source host or create a new one.
+
+    To redirect config output from the existing environment, use `pbm config >> pbm_config.yaml`, then copy the resulting file to the new environment.
+
+    Here's an example of config file content for AWS S3 compatible storage:
+
+    ```yaml
+    storage:
+      type: s3
+      s3:
+        region: us-west-2
+        bucket: pbm-test-bucket
+        prefix: data/pbm/backup
+        credentials:
+          access-key-id: <your-access-key-id-here>
+          secret-access-key: <your-secret-key-here> 
+    ```
+    The prefix name is the artifact name that appears in the **Backup name** column, under <i class="uil uil-history"></i> **Backup > All Backups** page.
```

Comment:
I think we can add a screenshot here to make it clear what user should use as a prefix.